common/changes/office-ui-fabric-react/master_2019-02-20-22-30.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add aria-describedby prop to Combo Box and Spin Button",
+      "type": "patch"
```

Comment:
This will need to be `minor` since the API has changed. You'll also get a CI error since the public API has changed, requiring you to run `npm run update-api`.

Reply:
Fixed.